ui/app/mainui/activitycenter/popups/ActivityCenterPopup.qml

```diff
@@ -21,88 +18,17 @@ import "../stores"
 Popup {
     id: root
 
-   // NOTE: temporary enum until we have different categories on UI and status-go sides
-    enum ActivityCategory {
-        All,
-        Admin,
-        Mentions,
-        Replies,
-        ContactRequests,
-        IdentityVerification,
-        Transactions,
-        Membership,
-        System
-    }
-    property int currentActivityCategory: ActivityCenterPopup.ActivityCategory.All
-    property int adminCount: 0
-    property int mentionsCount: 0
-    property int repliesCount: 0
-    property int contactRequestsCount: 0
-    property int identityRequestsCount: 0
-    property int membershipCount: 0
+    // FIXME: counters from service
```

Comment:
This piece is still draft

Reply:
When do you expect to put out of draft? In another PR?

Reply:
today)

Reply:
ready